### PR TITLE
Skeleton App

### PR DESCRIPTION
**Pull request by peteclark-ft**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      - store_test_results:
+          path: /tmp/test-results
+  dockerfile:
+    working_directory: /enriched-concepts
+    docker:
+      - image: docker:1.12.6-git
+    steps:
+      - checkout
+      - setup_docker_engine
+      - run:
```
Comment (izzyblues) [inline .circleci/config.yml]: Does it make sense make a check that the app starts in a docker container? e.g. by hitting the health check?

File: api/dredd-hooks.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+var hooks = require('hooks');
```
Comment (izzyblues) [inline api/dredd-hooks.js]: Why not in golang?

Comment (peteclark-ft) [reply api/dredd-hooks.js]: I don't see any real reason to - Dredd is a javascript service, and writing the hooks in go would add time on to the build (as it needs to run two processes and communicate between them) and increase the complexity (as the go libs are 3rd party).

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          name: Build Dockerfile
+          command: docker build -t test/enriched-concepts .
+      - run:
+          name: Run and Test Container
+          command: |
+            export CONTAINER_HASH=$(docker run -d -p 8080:8080 test/enriched-concepts)
```
Comment (izzyblues) [inline .circleci/config.yml]: I am a bit confused... You are telling me that it is not possible to make a curl request outside the enriched-concepts container?

Comment (peteclark-ft) [reply .circleci/config.yml]: Long story short... Nope!

**Pull request merged**